etc/helm/pachyderm/Chart.yaml

```diff
@@ -21,7 +21,7 @@ type: application
 # each time you make changes to the chart and its templates, including
 # the app version.  Versions are expected to follow Semantic
 # Versioning (https://semver.org/)
-version: 2.1.0
+version: 2.1.1
```

Comment:
is this necessary? I think we also want it to match appVersion

Reply:
Yup! ‘This version number should be incremented each time you make changes to the chart and its templates, including the app version.’ This change updates the chart, so the chart version needs to be incremented.

The chart version and the app version are two different things, both monotonically increasing. Increasing the app version necessitates increasing the chart version, but not vice-versa. E.g. correcting a buggy template does not necessarily mean a new release of Pachyderm itself (does it?).

Reply:
it's something we do at release time. You shouldn't need to increment it in the PR. Basically we keep version=appVersion. So every release increments this. Ex: https://github.com/pachyderm/pachyderm/blob/v2.2.1/etc/helm/pachyderm/Chart.yaml

Reply:
Righto! Reverted.